src/environment/player/action_parser.py

```diff
@@ -1,9 +1,11 @@
 from typing import Optional, List
 from environment.types import Direction
 from environment.player_action import PlayerAction
-from environment.player_action_move import PlayerActionMove
+from environment.player.action_move import ActionMove
+from environment.player.action_invalid import ActionInvalid
+from environment.player.action_accuse import ActionAccuse
 
-class PlayerActionParser():
+class ActionParser():
     @staticmethod
     def parse(action: str) -> Optional[PlayerAction]:
         if action.startswith("go"):
```

Comment:
Tu inak, ked pozeram na tento zapraseny kod 😄 ako sa to sem dostalo? 🥲 

Pouzil by som to co uz mame a jedna zmena. Nie je potreba deklarovat `direction = None`, priradzovat v nechutnom if-else a potom vratit. Mozeme bez vsetkeho jednoducho vratit. Nieco na tento styl:

```py
if action.startswith("go"):
    direction_text = action.split(" ")[1]
    for dir_enum in Direction:
        if dir_enum.name == direction_text.upper(): #uppercase text so that it matches with prop name of Direction
            return ActionMove(dir_enum.value)
            
    # handle wrongly typed direction